### DISCUSSION TITLE
3 PV Inverters

### DISCUSSION THREAD

**Discussion by RichisTesla** [ulic75/power-flow-card#123] [Q&A]
Hi, First i love Power Flow Card. My Problem now is that i dont know how to add my 2 new pv inverters so Power Flow Card can Show me in the top bubble Chart the power of alle 3 inverters together.

**Comment by kpfleming** [comment]
You need to make a 'template sensor' in Home Assistant to combine them together, and then display that in the Power Flow card.

**Comment by RichisTesla** [comment]
Thank you for your fast answer, i Hope i can do that 😬 im very new to Homeassistant an all that stuff here 😅

**Comment by kpfleming** [comment]
Oh, actually... if you are running HA 2022.12, you don't even need to create a `template sensor`, it's now possible to create it in the web interface. Go to `Settings` -> `Devices and Services` -> `Helpers` -> `Create Helper` -> `Combine the states of several sensors`.

**Comment by RichisTesla** [comment]
Big thank you, got it to work with template sensor before u told me to do it with create helper. Did it with helper again and it worked too. Learned a lot 😊 thanks again 👍
